.github/workflows/build-nym-vpn-app-windows-ev-sign.yml

```diff
@@ -218,16 +224,46 @@ jobs:
           rm public/licenses-rust.json || true
           npm run gen:licenses:rust
 
-      # - name: Import signing certificate
-      #   env:
-      #     WINDOWS_CERTIFICATE: ${{ secrets.WINDOWS_SIGNING_PFX_BASE64 }}
-      #     WINDOWS_CERTIFICATE_PASSWORD: ${{ secrets.WINDOWS_SIGNING_PFX_PASSWORD }}
-      #   run: |
-      #     New-Item -ItemType directory -Path certificate
-      #     Set-Content -Path certificate/tempCert.txt -Value $env:WINDOWS_CERTIFICATE
-      #     certutil -decode certificate/tempCert.txt certificate/certificate.pfx
-      #     Remove-Item -path certificate -include tempCert.txt
-      #     Import-PfxCertificate -FilePath certificate/certificate.pfx -CertStoreLocation Cert:\CurrentUser\My -Password (ConvertTo-SecureString -String $env:WINDOWS_CERTIFICATE_PASSWORD -Force -AsPlainText)
+      - name: Download EV CodeSignTool from ssl.com
+        working-directory: nym-vpn-app/src-tauri
+        shell: bash
+        run: |
+          curl -L0 https://www.ssl.com/download/codesigntool-for-linux-and-macos/ -o codesigntool.zip
+          unzip codesigntool.zip
+
+      - name: Get EV certificate credential id
+        working-directory: nym-vpn-app/src-tauri
+        id: get_credential_ids
+        shell: bash
+        run: |
+          echo "SSL_COM_CREDENTIAL_ID=$(./CodeSignTool.sh get_credential_ids -username=${{ secrets.SSL_COM_USERNAME }} -password=${{ secrets.SSL_COM_PASSWORD }} | sed -n '1!p' | sed 's/- //')" >> "$GITHUB_OUTPUT"
+
+      # Credential_id is obtained with ./CodeSignTool.sh get_credential_ids -username=${{ env.SSL_COM_USERNAME }} -password=${{ env.SSL_COM_PASSWORD }} | sed -n '1!p' | sed 's/- //'
+      # Absolute path to CodeSignTool is required since the tauri signcommand does not execute in the same context as the shell: bash for the step
+      # -override required to override the existing file
+      - name: Add custom sign command to tauri.conf.json
+        working-directory: nym-vpn-app/src-tauri
+        shell: bash
+        run: |
+          yq eval --inplace '.bundle.windows +=
+          {
+            "signCommand": {
+              "cmd": "C:\Program Files\Git\bin\bash.EXE",
+              "args": [
+                "/c/actions-runner/_work/nym-vpn-client/nym-vpn-client/nym-vpn-app/src-tauri/CodeSignTool.sh",
```

Comment:
Wonder if we can rely on `$GITHUB_WORKSPACE` to obtain the `/c/actions-runner/_work/nym-vpn-client/` part and AFAIK even if it's windows path, bash.exe should probably understand it right?

Reply:
No we can't.

`${{ github.workspace }}` give a path in the form of `C:\action-runner\_work\nym-vpn-client` since we're in windows.

Since we're running in bash, \n get interpreted as a line return.